helpers/queues.py

```diff
@@ -0,0 +1,81 @@
+"""
+Queues Helper class
+"""
+import asyncio
+import os
+import sys
+import boto3
+from botocore.exceptions import ClientError
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from conf import aws_credentials as awsc, queues_conf as qc
+
+class Queues():
```

Comment:
This is written as a class but really all these methods could be standalone. A better way is to create an `__ini__()` and create a self.client().